lib/Twig/Test/IntegrationTestCase.php

```diff
@@ -193,7 +193,7 @@ protected function doIntegrationTest($file, $message, $condition, $templates, $e
 
             if (false !== $exception) {
                 list($class) = explode(':', $exception);
-                $this->assertThat(null, new PHPUnit_Framework_Constraint_Exception($class));
+                $this->assertThat(null, new \PHPUnit\Framework\Constraint\Exception($class));
```

Comment:
this one should check whether the class exists, and fallback to the old one. The namespaced TestCase is available in PHPUnit 4.8.35 and 5.4+, but the exception is not